Python/specialize.c

```diff
@@ -412,6 +412,11 @@ _Py_Quicken(PyCodeObject *code) {
     return 0;
 }
 
+static inline int
+initial_counter_value(void) {
+    /* Choose a prime number ~50 */
```

Comment:
Does it make sense to keep the explanation from the now-removed `saturating_start`?

```suggestion
    /* Starting value for the counter.
     * Technically this should be 1, but that is likely to
     * cause a bit of thrashing when we optimize then get an immediate miss.
     * We want to give the counter a change to stabilize, so we start at
     * a prime number close to 50.
     */
```